packages/itwin/map-layers/src/ui/widget/MapUrlDialog.scss

```diff
@@ -83,8 +79,45 @@ $default-font-size: --iui-font-size-1;
   white-space: pre-wrap;
 }
 
+
+.map-layer-source-footer {
+  display: flex;
+  margin-top: 10px;
+  }
```

Comment:
Nitpick: spacing could be more consistent in this file

Reply:
@eringram done.